src/index.js

```diff
@@ -66,6 +69,14 @@ class Dropzone extends React.Component {
     document.body.onfocus = null
   }
 
+  composeHandlers(handler, target) {
```

Comment:
Since this is only called in the constructor and it returns `null`, this won't react to the change of the prop. We'll need to add a test for this (change from disabled to enabled state and check that events are being fired).